.gitignore

```diff
@@ -36,7 +36,7 @@ editor/benches/resources/50000_lines.roc
 editor/benches/resources/500_lines.roc
 
 # file editor creates when no arg is passed
-new-roc-project
+roc-projects
```

Comment:
The editor now creates `roc-projects/new-roc-project-1` which is to avoid issues with copying over the platform-switching example and the relative imports in the platform it uses.